doc/nrf/ug_matter_configuring_controller.rst

```diff
@@ -24,27 +24,24 @@ The following figure shows the available Matter controllers in the |NCS|.
 
 The Matter controller implementation for the nRF Connect platform can be of the following types:
 
-* Python controller for Linux
+* CHIP Tool controller for Linux or macOS (recommended)
```

Comment:
Doesn't it look like we recommend the macOS version? I mean the recommendation concerns the chip-tool in general.

Reply:
Rephrased.